js/app/page.tsx

```diff
@@ -220,7 +279,7 @@ function NavBar() {
                 onClick={() => {
                   setLocation(href);
                 }}
-                isDisabled={isLoading || isFlagLoading || disable}
+                isDisabled={(isLoading ?? isFlagLoading) || disable}
```

Comment:
Hi Kent,

I think we should not refer `isFlagLoading` when `isLoading` is undefined.
Should we chang to `{isDisabled={!!isLoading || isFlagLoading || disable}}` to fix the lint warning?

Reply:
Fixed!